static/app/views/replays/utils/useReplayEvent.tsx

```diff
@@ -215,6 +220,7 @@ function useReplayEvent({eventSlug, location, orgId}: Options): Result {
     onRetry,
 
     breadcrumbEntry: state.breadcrumbEntry,
+    consoleMessages: state.consoleMessages,
```

Comment:
wonder if we should always return the entirety of `state` here to reduce boilerplate.

Reply:
Do we need any of these fields in state at all? They're derived from `event, rrwebEvents, replayEvents` so maybe having only those three in state is all that is required.

I will answer that question, doesn't look like too many conflicting PR's are out right now

Reply:
So turns out we don't need as much state.

The change in https://github.com/getsentry/sentry/pull/34194 still makes sense, but we can move these filter/calculation calls down closer to the `return` statement and skip putting them in state.

I'm setting up a PR, probably best to merge the changes slowly to avoid any git conflicts.

Reply:
I'll wait for the above PRs to be merged first